src/worldcereal/openeo/feature_extractor.py

```diff
@@ -62,6 +63,25 @@ def output_labels(self) -> list:
         of the presto embeddings"""
         return [f"presto_ft_{i}" for i in range(128)]
 
+    def _compute_slope(self, inarr: xr.DataArray) -> xr.DataArray:
+        """Computes the slope using the richdem library. The input array should
+        have the following bands: 'elevation' And no time dimension. Returns a
+        new DataArray containing the new `slope` band.
+        """
+        from richdem import (  # pylint: disable=import-outside-toplevel
+            TerrainAttribute,
+            rdarray,
+        )
+
+        dem = inarr.sel(bands="elevation").values
+        dem_array = rdarray(dem, no_data=65535)
```

Comment:
have you tested if the `no_data` value is handled correctly? Should we add a test for this to be sure?

Reply:
I remember that you asked me to verify this but I ended up dismissing it because unlike in the satio wrapper, here you can specify the `no_data` parameter.

You did good to ask because I checked and indeed it does accepts 65535 as nodata, but the output still uses -9999...

I added a fix as well as a small  test for this.

Reply:
OK nice!